### PR TITLE
dtspo182: Added DNS records for toffee App

### PR DESCRIPTION
**Pull request by mokainos**
https://tools.hmcts.net/jira/browse/DTSPO-182
Added DNS records for toffee App

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: environments/sandbox.yml [hunk 1]
```diff
@@ -126,3 +126,19 @@ cname:
...
     ttl: 300
     record: "cdnverify.hmcts-bulkscan-shutter-sbox.azureedge.net"
+  - name: “toffee”
+    ttl: 300
+    record: "sdshmcts-sbox.azurefd.net"
+    shutter: false
+  - name: "www.toffee"
+    ttl: 300
+    record: "sdshmcts-sbox.azurefd.net"
+  - name: "afdverify.plum"
```
Comment (timja) [inline environments/sandbox.yml:136]:
```suggestion
  - name: "afdverify.toffee"
```

File: environments/sandbox.yml [hunk 2]
```diff
@@ -126,3 +126,19 @@ cname:
   - name: "cdnverify.bulkscan"
     ttl: 300
     record: "cdnverify.hmcts-bulkscan-shutter-sbox.azureedge.net"
+  - name: “toffee”
+    ttl: 300
+    record: "sdshmcts-sbox.azurefd.net"
+    shutter: false
+  - name: "www.toffee"
```
Comment (timja) [inline environments/sandbox.yml:133]: we won't have an ssl cert for this so shouldn't have the www

Comment (mokainos) [reply environments/sandbox.yml]: removed both entries (www and afdverfy.www)

File: environments/sandbox.yml [hunk 3]
```diff
@@ -126,3 +126,13 @@ cname:
...
     ttl: 300
     record: "cdnverify.hmcts-bulkscan-shutter-sbox.azureedge.net"
+  - name: “toffee”
+    ttl: 300
+    record: "sdshmcts-sbox.azurefd.net"
+    shutter: false
+  - name: "afdverify.toffee"
+    ttl: 3600
+    record: "afdverify.sdshmcts-sbox.azurefd.net"
+  - name: "toffee.uksouth"
```
[the remark this answers is not in the document]
Comment (mokainos) [reply environments/sandbox.yml]: removed it.

**Review by luigibk — APPROVED** [review]

File: environments/sandbox.yml [hunk 4]
```diff
@@ -126,3 +126,10 @@ cname:
   - name: "cdnverify.bulkscan"
     ttl: 300
     record: "cdnverify.hmcts-bulkscan-shutter-sbox.azureedge.net"
+  - name: “toffee”
+    ttl: 300
+    record: "sdshmcts-sbox.azurefd.net"
+    shutter: false
+  - name: "afdverify.toffee"
```
Comment (timja) [inline environments/sandbox.yml:133]: also need cdnverify records as well

https://github.com/hmcts/azure-public-dns/blob/features/dtspo182/environments/sandbox.yml#L87-L89

should be this I think:

cdnverify.sdshmcts-toffee-shutter-sbox.azureedge.net

**Pull request merged**